elasticsearch/client.go

```diff
@@ -63,16 +73,13 @@ func (c Client) Index(i Indexable) *IndexResult {
 	req.Header.Add("Content-Type", "application/json")
 
 	// Sign the request, send it, and print the response
-	_, err = signer.Sign(req, body, service, region, time.Now())
-	if err != nil {
-		c.logger.Println(err.Error())
-	}
+	_, _ = signer.Sign(req, body, service, region, time.Now())
```

Comment:
Do we not care if an error occurs here? If not, you could just do `signer.Sign(req, body, service, region, time.Now())` without the assignment.

Reply:
I was thinking that if signing fails the request should also fail with an appropriate error message, which is handled and logged further down 🤷  happy to revert it though